src/components/PetsList.jsx

```diff
@@ -1,19 +1,87 @@
 import FilterList from "./FilterList";
 import PetCard from "./PetCard";
 import { PetItems } from "../files/PetsData";
-import React from "react";
+import React, { useState } from "react";
+import CategoriesFilterCollapse from "./CategoriesFilterCollapse";
 
 const PetsList = () => {
+  const [filteredItems, setFilteredItems] = useState([]);
+  const [isFiltering, setIsFiltering] = useState(false);
```

Comment:
@tmccoy529 didn't we have a hook for something like this with filters?

Reply:
@martypowell we did do that for newsroom to load more since it was an endpoint call. Service list just breaks this logic out into a separate file for readability. Without an endpoint not sure a hook would be necessary.